src/js/common/schemaform/SchemaForm.jsx

```diff
@@ -56,6 +56,8 @@ class SchemaForm extends React.Component {
       this.setState({ formContext: _.set('pageTitle', newProps.title, this.state.formContext) });
     } else if (!!newProps.reviewMode !== !!this.state.formContext.reviewMode) {
       this.setState(this.getEmptyState(newProps));
+    } else if (!!newProps.prefilled !== !!this.state.formContext.prefilled) {
```

Comment:
Yeah, honestly, it's not really necessary in these if statements. It's more relevant if you're setting a value.